libqtile/backend/base.py

```diff
@@ -325,21 +325,10 @@ def match(self, match: config.Match) -> bool:
         return match.compare(self)
 
     @abstractmethod
+    @expose_command()
     def focus(self, warp: bool) -> None:
```

Comment:
should `Static.focus` be exposed?

Reply:
At the minute they can't be focussed (i.e. `qtile.current_window` will never return a static window). They need some work so that they can be focussed (and also made un-static, among other things). But ATM no